src/js/lib/client/src/index.js

```diff
@@ -332,28 +332,47 @@ Electron.app.on('web-contents-created', (event,contents) => {
 // === Project Manager ===
 // =======================
 
-async function withBackend(opts) {
+function projectManagerPath() {
     let binPath = args['backend-path']
     if (!binPath) {
         binPath = paths.get_project_manager_path(resources)
     }
     let binExists = fss.existsSync(binPath)
     assert(binExists, `Could not find the project manager binary at ${binPath}.`)
+    return binPath
+}
+
+async function execProjectManager(args) {
+    let binPath = projectManagerPath()
+    return await execFile(binPath,args).catch(function(err) {throw err})
+}
 
-    let out = await execFile(binPath,opts).catch(function(err) {throw err})
+function spawnProjectManager(args) { 
+    let binPath = projectManagerPath()
+    let stdin = 'pipe'
+    let stdout = 'inherit'
+    let stderr = 'inherit'
+    let opts = {
+        stdio: [stdin,stdout,stderr]
+    }
+    let out = child_process.spawn(binPath,args,opts)
+    console.log(`Project Manager has been spawned, pid = ${out.pid}.`) 
+    out.on('exit', (code) => {
+        console.log(`Project Manager exited with code ${code}.`)
+    })
```

Comment:
Please write docs here what this does. It is not obvious from the code and without docs its error prone regarding future refactoring. I don't understand why this fixes this issue and this should be explained in the docs for future reference.